spot_data_responses.go

```diff
@@ -104,6 +104,36 @@ type SingleMarketStatistics struct {
 	Data TickerData
 }
 
+// AllMarketStatistics
```

Comment:
These structs are wrong,
You should define Data structures like JSON responses.
I've sent you more details on Telegram.